doc/api/net.markdown

```diff
@@ -413,7 +413,8 @@ Returns `socket`.
 ### socket.remoteAddress
 
 The string representation of the remote IP address. For example,
-`'74.125.127.100'` or `'2001:4860:a005::68'`.
+`'74.125.127.100'` or `'2001:4860:a005::68'`. Value may be `undefined` if
+socket is destroyed (client disconnected, ...).
```

Comment:
The parenthetical seems confusing more than clarifying. I would say either remove it, or change it to be more explicit: `(for example, if the client disconnected)`